karton/misp_pusher/misp_pusher.py

```diff
@@ -64,10 +64,11 @@ def process(self, task: Task) -> None:
             # Nothing actionable found - skip the config
             return
 
-        misp = ExpandedPyMISP(
-            http_url(self.config.get("misp", "url")),
-            self.config.get("misp", "key"),
-            not self.config.getboolean("misp", "insecure", False),
+        misp = PyMISP(
```

Comment:
I don't remember what's the difference between ExpandedPyMISP and PyMISP but I hope it works :thinking:

Reply:
`ExpandedPyMISP` has been deprecated and users should use `PyMISP`